R/page_size.R

```diff
@@ -48,6 +48,7 @@ pg_dim_names <- list(
 #' @return for `page_types` a character vector of supported page types,
 #' for `page_dim` the dimensions (width, then height) of the selected page type.
 #'
+#' @keyword internal
 #' @export
```

Comment:
One should be selected

Reply:
Same for other internal calls

Reply:
Thanks man, I should have made this a draft PR, it was working progress, I realize the problem yesterday when I was making changes. let me get back to you on this. I think a few of these functions can be made internal, won't have to be exported. let me chck that

Reply:
I don't think this should be internal because it is how the user can find out what page types we support.